fibonicciSeries/FibonicciSeriesProgram.java

```diff
@@ -0,0 +1,21 @@
+package fibonicciSeries;
+
+/**
+ * Fibonacci Series program in Java
+ */
+public class FibonicciSeriesProgram {
+    public static void main(String args[]) {
+        fibonacciSeries(10);
+    }
+
+    public static void fibonacciSeries(int count) {
+        int firstNumber = 0, secondNumber = 1, nextNumber, index;
+        System.out.print(firstNumber + " " + secondNumber);
+        for (index = 2; index < count; ++index) {
```

Comment:
Can you change the logic. Use recursion.

Reply:
Used recursion